lib/components/Stepper/stepper.scss

```diff
@@ -0,0 +1,105 @@
+@function calculate-margin-top($node-size, $spacing, $line-height) {
+    @return calc((#{$node-size} + (#{$spacing} * 2) - #{$line-height}) / 2);
+}
+
+@mixin margin-top-for-size($size) {
+    margin-top: calculate-margin-top(var(--component-stepper-node-size-#{$size}),
+        var(--component-stepper-spacing-3xs),
+        var(--semantic-typography-body-#{$size}-bold-default-lineHeight)) !important;
+}
+
+.steps-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.step {
+    display: flex;
+    gap: 1rem;
+
+    &-circle {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        box-sizing: border-box;
+        -moz-box-sizing: border-box;
+        -webkit-box-sizing: border-box;
+
+        border-radius: var(--component-stepper-node-border-radius);
+        border: var(--component-stepper-node-border-width) solid var(--component-stepper-node-border-color-active);
+        margin: var(--component-stepper-spacing-3xs) 0;
+
+        &__size-sm {
+            width: var(--component-stepper-node-size-sm);
+            height: var(--component-stepper-node-size-sm);
+        }
+
+        &__size-md {
+            width: var(--component-stepper-node-size-md);
+            height: var(--component-stepper-node-size-md);
+        }
+
+        &__size-lg {
+            width: var(--component-stepper-node-size-lg);
+            height: var(--component-stepper-node-size-lg);
+        }
+
+        &--active {
+            background-color: var(--component-stepper-node-border-color-active);
+        }
+
+        &--inactive {
+            background: none;
+        }
+
+        &--disabled {
+            border-color: var(--component-stepper-node-border-color-incomplete);
+        }
+
+        &__pointer {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+        }
+
+        &__label {
+
+            &--disabled {
+                color: var(--component-stepper-node-border-color-incomplete) !important;
+            }
+
+            &-sm {
+                @include margin-top-for-size(sm);
+            }
+
+            &-md {
+                @include margin-top-for-size(md);
+            }
+
+            &-lg {
+                @include margin-top-for-size(lg);
+            }
+        }
+    }
+
+    &-line {
+        width: var(--component-stepper-track-width);
+        background-color: var(--component-stepper-node-border-color-active);
+
+        &--disabled {
+            background-color: var(--component-stepper-node-border-color-incomplete);
+        }
+
+        &--size-sm {
+            height: 8px;
+        }
+
+        &--size-md {
+            height: 16px;
+        }
+
+        &--size-lg {
+            height: 26px;
+        }
```

Comment:
I think designer are using same line height in the figma for sm, md and lg. They separate it to two line which is 8px + 8px but when we have label + description the line height change to 26px. Can you double check on this with designer I think we don't have a token for component stepper we don't have 26px, you'll need to use core in this case. 

```suggestion
        height: calc(var(--component-stepper-track-height-md) + var(--component-stepper-track-height-md))
```

Reply:
I thought it might be helpful to attach a screenshot here for better context. From what I can see, it looks like the designer is using 8px + 8px across sm, md, and lg. I haven't come across any other size combinations like 8px, 16px or 26px, but please feel free to correct me if I'm wrong!
<img width="1028" alt="image" src="https://github.com/user-attachments/assets/8d4ae20a-a0f0-42da-b3e7-4b568840c892">

Reply:
@jia-deriv 
small size is 8px in total
middle size is 16px
large size is 26px
but if I apply just this formula it doesn't work as expected
height: calc(var(--component-stepper-track-height-**sm**) + var(--component-stepper-track-height-**sm**))
height: calc(var(--component-stepper-track-height-**md**) + var(--component-stepper-track-height-**md**))
height: calc(var(--component-stepper-track-height-**lg**) + var(--component-stepper-track-height-**lg**))

Reply:
If that’s the case, let's double check with the designer to see if the tokens might be missing.

Reply:
good point🧐 right👍
I've requested it here https://deriv-group.slack.com/archives/C054V5QFXD3/p1728045249311469?thread_ts=1727873914.297319&cid=C054V5QFXD3
thank you🙇

Reply:
1)sm left as it is
2)lg size I removed since there should be a different type of flexible calculation depending on the description height, we can do it later

Reply:
Sure, thank you @maryia-matskevich-deriv